docs/developers-guide/email-templates.md

```diff
@@ -0,0 +1,12 @@
+### Email templating
+
+We use golang templates for email templating.
+templates located in `backend/templates` and embeded into binary by statik
+
+For getting access to files you can use package `templates` from `backend/app/templates`
```

Comment:
It would be nice to rephrase that or add separate paragraph about how to replace these templates with your own. Also, I would suggest moving templates to `backend/templates` removing them from `app`.

Reply:
Per offline talk, no ability to replace templates now.